book/website/ethical-research/activism/activism-case-study-google.md

```diff
@@ -0,0 +1,53 @@
+(er-activism-case-study-google)=
+# Case study: Google Workers & Project Maven
+
+Tech workers have historically had low levels of unionisation for a variety of different reasons. 
+Some have also been ambivalent about the potential impact of the companies and products they work for on systemic inequality and injustice across the globe.
+
+However, more and more people who work in the industry are speaking up and demanding that companies and organisations consider the ethics and impact of the work they do and who they do that work with. 
+
+Google is one example of a large tech company where workers have spoken up on several different issues including internal company policy and external contracts. 
+
+(er-activism-case-study-google-advocacy)=
+## Advocacy & Workplace Activism by Google Workers 
+### Project Maven - 2018
+One example of this is Google workers campaigning against Project Maven, a contract with the US Pentagon to use AI to develop more effective drone guidance.
```

Comment:
I think here 'this' is referring to 'workplace activism' ? May just improve readability to state it explicitly as it sounds as though it had followed on from a previous sentence rather than new section. 

```suggestion
One example of workplace activism is Google workers campaigning against Project Maven, a contract with the US Pentagon to use AI to develop more effective drone guidance.
```